docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,130 @@
+
+Identifying and writing down these problems also helps us write our objectives and key results because the objectives are created to list **what we want to do** based on the problem we are trying to solve, and the key results show **how we will measure that it’s been done**.
+
+Each team in Gatsby has objectives and key results that they are responsible for hitting.
+
+The product organization is responsible for making sure our key results are set and that any dependencies on other teams are clear, so they can successfully achieve their key results as well.
+
+### Creating an outcome-based roadmap that gives the company a sense of where we are with product development or features
+
+As we learn more about our community and our customers through research, discovery, and metrics, we will create a roadmap that shows what outcomes we want to provide for people. This is different from a traditional roadmap because we won’t be focusing on features, but rather on the problems we’ve seen and the outcome that would show we’ve solved the problem.
+
+### Giving internal stakeholders high integrity commitments
+
+There is always some measure of conflict between sales, marketing, engineering, and product. Each group has a goal that it’s trying to accomplish. These goals should create a checks and balances system that keeps teams researching, prototyping, building, and communicating with customers. As an example, engineering wants to build the best possible product from a technical standpoint. Product wants to create value for the users. Sales wants to sell as much of the product as possible to the broadest audience on the shortest timeline. Marketing wants exciting improvements that generate buzz and create opportunities for publicity.
+
+All of these goals are important, but create a necessary tension that forces us to make collaborate and make decisions in order to keep moving forward.
+
+As we create and improve upon our products, managing that conflict works best when expectations are properly set.
+
+Because we believe that setting timelines creates an arbitrary end date, we instead aim to create very actionable and small scopes of work that we can tie to a **high integrity commitment**.
+
+A high integrity commitment is a commitment that we can make to internal stakeholders about when an iteration, product, or feature will be complete.
+
+Instead of giving this outright, we instead look at the problem we’re trying to solve, do prototyping and feasibility testing, and create a scope. After taking the time to understand what the problem is, how we can solve it, and how much time it will take us to create the solution, we’re able to give a much better estimate for highly awaited products or features.
+
+We won’t give these high-integrity commitments for everything, but if we know that sales or marketing is waiting on us to move forward, then we have a reasonable timeframe to talk about that’s based in reality.
```

Comment:
I got a little lost in this section. It took me a minute to make the link between a high integrity commitment and setting expectations to manage conflict.

Reply:
I think this section also seems less relevant to OSS users in the way that it includes details about how this works internally but not externally (e.g. people probably just want to know "when will themes be ready?" and this doesn't give a straight answer about that...imho.

Reply:
That's a good point. I also think we should be giving high integrity commitments to our community, too, when we have them. I'll make a few updates to this section. Thanks!

Reply:
@shannonbux: What do you think of this addition. I think that it helps add more context and explanation of how we're going to use this externally with our community as well.

"This will also help us set better expectations with our community, and make our plans more clear."

Reply:
Saw the other comment about wanting more examples, and I think we'll need to document examples as we have them. I meant this to be mostly informational, but agree that having examples whenever possible is great!